torch/csrc/jit/codegen/cuda/interface.cpp

```diff
@@ -5,13 +5,45 @@
 #include <torch/csrc/jit/runtime/custom_operator.h>
 #include <torch/csrc/jit/runtime/register_ops_utils.h>
 
+// NOLINTNEXTLINE
+C10_DEFINE_bool(
+    torch_jit_nvfuser_singleton_fusion,
+    false,
+    "enable single node fusion for nvfuser");
+
+// NOLINTNEXTLINE
+C10_DEFINE_bool(
+    torch_jit_nvfuser_horizontal_fusion,
+    true,
+    "enable single node fusion for nvfuser");
```

Comment:
Shouldn't this comment be to enable Horizontal fusion?

Reply:
Oops, copy pasted :stuck_out_tongue: